syntax = "proto2";

package Super;

message ServerEntry
{
	optional uint32 id = 1;
	optional uint32 type = 2;
	optional string name = 3;
	optional string ip = 4;
	optional uint32 port = 5;
	optional string extip = 6;
	optional uint32 extport = 7;
	optional uint32 nettype = 8;
}

message t_Startup_Request
{
	optional uint32 type = 1;
}

message t_Startup_Response
{
	//required ServerEntry entry = 1;
	required uint32 index = 1;
	repeated ServerEntry entry = 2;
}

//login to game server
message stLoginToGame
{
	optional string account = 1;
	optional string key = 2;
}

message stLoginGameServerResult
{
	optional uint32 ret = 1;
}

message stSearchFight
{
}
